src/schema/slack.ts

```diff
@@ -0,0 +1,153 @@
+import { IResolvers } from '@graphql-tools/utils';
+import { Context } from '../Context';
+import { traceResolvers } from './trace';
+import { WebClient } from '@slack/web-api';
+import { logger } from '../logger';
+import { getLimit, getSlackIntegrationOrFail } from '../common';
+import { GQLEmptyResponse } from './common';
+import { ValidationError } from 'apollo-server-errors';
+import { UserSourceIntegrationSlack } from '../entity/UserSourceIntegration';
+import { Source } from '../entity';
+
+export type GQLSlackChannels = {
+  id?: string;
+  name?: string;
+};
+
+export const typeDefs = /* GraphQL */ `
```

Comment:
Should we make the terms here more generic to support future integrations?
Should we consider changing the file name as well?

Reply:
I am not sure there is gonna be the same calls for other integrations but maybe `integrations` schema makes more sense yes.